app/views/TeamChannelsView.tsx

```diff
@@ -320,23 +304,27 @@ class TeamChannelsView extends React.Component<ITeamChannelsViewProps, ITeamChan
 
 	goRoomActionsView = (screen: string) => {
 		logEvent(events.TC_GO_ACTIONS);
-		const { team } = this;
+		const { team, joined } = this;
 		const { navigation, isMasterDetail } = this.props;
-		if (isMasterDetail) {
+		if (!team) {
+			return;
+		}
+		if (isMasterDetail && screen) {
 			navigation.navigate('ModalStackNavigator', {
-				screen: screen ?? 'RoomActionsView',
+				screen: 'RoomActionsView',
 				params: {
 					rid: team.rid,
 					t: team.t,
 					room: team,
-					showCloseModal: false
```

Comment:
Sure you could remove this property?

Reply:
Yes, we now send `navigation` directly to `Header.CloseModal`